### ISSUE TITLE
Feature request: auto class import

### ISSUE THREAD

**Issue by kevinresol** [snowkit/atom-haxe#36]
Reference the functionality in FD:
-  Type the class name directly and press `ctrl+shift+1`, the class will be imported automagically. If there are more than one class of the same name, allow user to choose.
  
  > ![capture1](https://cloud.githubusercontent.com/assets/1103788/8866806/3748b8ac-31f3-11e5-9c0a-5a9fbcd7cf37.PNG)
-  Autocomplete + autoimport whenever a class is expected (e.g. `var a:|` or `function():|`).
  
  > ![capture2](https://cloud.githubusercontent.com/assets/1103788/8866814/5cee9ee6-31f3-11e5-96d9-adeb6d3f5830.PNG)

**Comment by jeremyfa** [comment]
Yes, this would be a nice feature to have. Probably just a matter of _when_ we will add it :)
